fixtures/pages-functions-wasm-app/tests/index.test.ts

```diff
@@ -3,7 +3,7 @@ import { fetch } from "undici";
 import { describe, it, beforeAll, afterAll } from "vitest";
 import { runWranglerPagesDev } from "../../shared/src/run-wrangler-long-lived";
 
-describe.concurrent("Pages Functions with wasm module imports", () => {
+describe("Pages Functions with wasm module imports", () => {
```

Comment:
Removing the `concurrent` from all the test suites not only was necessary for TurboRepo (it tries to parallelize tests without dependencies) but the tests were passing consistently (I ran a battery script on them) without it.